migrations/20190410101237-fix-localField-type.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+    async up (db) {
+        var operations = [];
+
+        const ObjectID = require('mongodb').ObjectID;
+        const reports = await db.collection('wst_Reports').find().toArray();
+        reports.forEach(report => {
+            var layerID = report.selectedLayerID;
+            operations.push({
+                'updateOne': {
+                    'filter': {'_id': report._id},
+                    'update': {
+                        '$set': {'selectedLayerID': ObjectID(layerID)}
+                    }
+                }
+
+            });
+        });
+
+        db.collection('wst_Reports').bulkWrite(operations);
```

Comment:
bulkWrite returns a Promise.
It should be returned by `up` so that migrate-mongo know when the bulkWrite is finished.

Reply:
The migration fails if there is no reports in mongo database.

```
UnhandledPromiseRejectionWarning: MongoError: Invalid Operation, no operations specified
    at Function.create (/home/urungi/urungi/node_modules/migrate-mongo/node_modules/mongodb-core/lib/error.js:43:12)
    at toError (/home/urungi/urungi/node_modules/migrate-mongo/node_modules/mongodb/lib/utils.js:149:22)
    at OrderedBulkOperation.bulkExecute (/home/urungi/urungi/node_modules/migrate-mongo/node_modules/mongodb/lib/bulk/common.js:937:31)
    at OrderedBulkOperation.execute (/home/urungi/urungi/node_modules/migrate-mongo/node_modules/mongodb/lib/bulk/ordered.js:117:22)
    at bulkWrite (/home/urungi/urungi/node_modules/migrate-mongo/node_modules/mongodb/lib/operations/collection_ops.js:115:8)
    at /home/urungi/urungi/node_modules/migrate-mongo/node_modules/mongodb/lib/utils.js:437:24
    at new Promise (<anonymous>)
    at executeOperation (/home/urungi/urungi/node_modules/migrate-mongo/node_modules/mongodb/lib/utils.js:432:10)
    at Collection.bulkWrite (/home/urungi/urungi/node_modules/migrate-mongo/node_modules/mongodb/lib/collection.js:595:10)
    at up (/home/urungi/urungi/migrations/20190410101237-fix-localField-type.js:21:38)
```

You should not call `bulkWrite` if `operations` is empty